media-sound/ario/ario-1.6-r1.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python2_7 )
+
+inherit autotools python-any-r1
+
+DESCRIPTION="yayy GTK client for MPD inspired by Rhythmbox but much lighter and faster"
```

Comment:
`"yayy"`? :) 
Is this suppose to mean something, or is it just a hooray? Maybe that can be taken out.